### PR TITLE
Expand access to TILES data

### LINKED ISSUE #217: Enable noise measurement and forced extraction to Stokes I tiles
As the noise and background maps are now available for Stokes I tile image.

### LINKED ISSUE #320: Enable access to Tiles RMS data
In the past RMS data didn't exist for TILES and therefore various checks were put into the `Query` class to ensure scripts exited gracefully if RMS tiles data was requested.

However, that data now exists and therefore we should be providing access to it.

To do:
* Update _add_files to calculate the correct rms_file path
* Remove validation statements in Query initialisation

### PR DESCRIPTION
**Pull request by ddobie**
Fix #320.
Fix #217.

### CODE REVIEW

**Review by ajstewart — COMMENTED** [review]
Is it worth a test to check that the stokes parameter is being understood and used correctly when it gets the files?

**Comment by ddobie** [comment]
I'm not sure what you mean by that

**Comment by ajstewart** [comment]
> I'm not sure what you mean by that

Sorry, I was meaning the test that has changed in this PR, the get_files one, it isn't testing that the different stokes files are obtained correctly. I wondered if it was worth adding a parametrize on that test to also add the stokes argument (if it works like that I can't remember - or if there isn't already a stokes test that I have forgotten about).

File: vasttools/query.py [hunk 1]
```diff
@@ -1713,17 +1706,19 @@ def _add_files(self, row: pd.Series) -> Tuple[str, str, str]:
             dir_name = "TILES"
 
             image_file_fmt = (
-                "image.i.{}.SB{}.cont"
+                "image.{}.{}.SB{}.cont"
                 ".taylor.0.restored.fits".format(
-                    row.field, row.sbid
+                    self.settings['stokes'].lower(), row.field, row.sbid
                 )
             )
```
Comment (ajstewart) [inline vasttools/query.py:1713]: I might be getting confused here, is there now access to stokes V tiles? And not just stokes I?

As my last comment came from this bit where the stokes has been added.

As there is also this test which may need to be sorted out:
https://github.com/askap-vast/vast-tools/blob/e6d17b78d2e3d5ee65981ac1071825953e769012/tests/test_query.py#L566-L594

Comment (ddobie) [reply vasttools/query.py:1713]: I'm also a tad confused, but I think/hope my latest commit has addressed what you're talking about.

You're right, there's no access to stokes V tiles yet, but I figured I might as well remove the hardcoded stokes I while I'm working on that part of the code so it's there and ready if we do decide to add it in the future.

Comment (ajstewart) [reply vasttools/query.py:1713]: Ok so the coded blocks for Stokes V tiles, like what that test I linked to above is checking, should remain for now?

**Review by ajstewart — DISMISSED** [review]
All good if the Stokes V tiles block does need to remain.

**Pull request merged**